### ISSUE TITLE
As a content editor, I can create lists in a 4 column grid, in addition to 2 and 3 column.

### ISSUE THREAD

**Issue by mark-bennett-uiowa** [uiowa/uiowa#3700]
# Story

As a content editor, I can create lists in a 4 column grid, in addition to 2 and 3 column grids. Currently, we can only select 2 and 3 column grid layouts in some widgets.

![Screen Shot 2021-06-22 at 1 19 39 PM](https://user-images.githubusercontent.com/3067017/122979122-cb86e780-d35c-11eb-9e56-45be0f125c11.png)

This can be seen on [https://cs.prod.drupal.uiowa.edu/research/languages](https://cs.prod.drupal.uiowa.edu/research/languages):

![Screen Shot 2021-06-22 at 1 22 06 PM](https://user-images.githubusercontent.com/3067017/122979209-e78a8900-d35c-11eb-8001-8f07c1042eef.png)

Can we add 4 column as an option? Maybe only with wide sections?

# Proposed solution

Create a 4 column grid option (that is only available in wide sections)?

**Comment by bspeare** [comment]
I think this is only possible if we decide to add grid options to the sections as an option that is in process here: https://github.com/uiowa/uiowa/pull/2707.    I think we wanted to have grid options available on the section level because there are already poorly defined grid options for `Layout: Grid` on the One column section. 

Adding a 4 column option on the block grid options becomes problematic because we wouldn't want it used outside of the One column section.

**Comment by mark-bennett-uiowa** [comment]
@bspeare Good point Ben and I agree. I think locking them down to one-column is essential, and possibly further locking 4 column down to Wide width, to avoid squash of content. I was worried that there was another concern with 4 column, so wanted to raise the issue to clarify my own understanding.

**Comment by bspeare** [comment]
@pyrello do you think we can still move forward with https://github.com/uiowa/uiowa/pull/2707 as a replacement for: 

- https://github.com/uiowa/uiowa/blob/c05630523a8d365227edb66a78d40195da167a2e/config/default/layout_builder_styles.style.section_layout_grid_1x3.yml
- https://github.com/uiowa/uiowa/blob/c05630523a8d365227edb66a78d40195da167a2e/config/default/layout_builder_styles.style.section_layout_grid_3x2.yml
- https://github.com/uiowa/uiowa/blob/c05630523a8d365227edb66a78d40195da167a2e/config/default/layout_builder_styles.style.section_layout_grid_3x2_stacked.yml

**Comment by pyrello** [comment]
Yes. There is still a lot of work to finish the implementation, but I think it is a viable path forward.

**Comment by briand44** [comment]
Somewhat related? #4317
